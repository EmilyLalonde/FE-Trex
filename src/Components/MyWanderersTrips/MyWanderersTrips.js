import React, { Component } from "react";
import FollowerHeader from '../FollowerHeader/FollowerHeader';
import FollowerFooter from '../FollowerFooter/FollowerFooter'
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
  TextInput,
  Keyboard
} from "react-native";

export default class MyWanderersTrips extends Component {
  constructor() {
    super() 
    this.state = {

    }
  }
  render() {
    const {navigate} = this.props.navigation;
    return (
      <View style={styles.container}> 
        <FollowerHeader />
        <ScrollView>
        <Text>Wanderer trip info</Text>
        </ScrollView>
        <FollowerFooter navigate={navigate}/>
      </View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
    alignItems: 'stretch',
    justifyContent: 'flex-start'
  }, 
  text: {
    color: 'white',
    marginVertical: 40,
    textAlign: 'center',
    fontSize: 30,
    width: 'auto'
  }, 
  button: {
    borderColor: 'white',
    borderWidth: 1,
    borderRadius: 8,
    borderStyle: 'solid',
    width: 'auto',
    height: 60,
    margin: 20,
    fontSize: 30,
    padding: 10,
    color: 'white',
    textAlign: 'center',
    backgroundColor: '#84183B',
    alignItems: 'stretch'
  }
});