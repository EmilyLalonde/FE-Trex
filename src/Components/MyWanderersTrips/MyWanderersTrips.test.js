import React from 'react';
import 'react-native';
import MyWanderersTrips from './MyWanderersTrips';
import { shallow } from 'enzyme';

describe('MyWanderersTrips', () => {
  it('should return true', () => {
    expect(true).toEqual(true)
  })

it('should match the snapshot when loading', () => {
  let wrapper = shallow(<MyWanderersTrips navigation={{ navigate: jest.fn() }} />)

  expect(wrapper).toMatchSnapshot();
})
})