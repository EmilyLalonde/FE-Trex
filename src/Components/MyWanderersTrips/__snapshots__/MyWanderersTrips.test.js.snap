// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWanderersTrips should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Header />
  <ScrollViewMock>
    <Text>
      Wanderer trip info
    </Text>
  </ScrollViewMock>
  <Footer
    navigate={[MockFunction]}
  />
</View>
`;
